mcs/build/profiles/monodroid.make

```diff
@@ -29,6 +29,9 @@ PROFILE_MCS_FLAGS = \
 	-nowarn:1699 \
 	-nostdlib \
 	$(DEFAULT_REFERENCES) \
+	-d:NO_SYSTEM_DRAWING_DEPENDENCY \
```

Comment:
I copied this from XM to get system.runtime.remoting working.
I'll move that to the right makefile then.